pkg/preflight/preflight.go

```diff
@@ -303,3 +304,47 @@ func CreateRenderedSpec(appID string, sequence int64, origin string, inCluster b
 
 	return nil
 }
+
+func injectDefaultPreflights(preflight *troubleshootv1beta2.Preflight, kotskinds *kotsutil.KotsKinds, registrySettings registrytypes.RegistrySettings) {
+	if !registrySettings.IsValid() || !registrySettings.IsReadOnly {
+		return
+	}
+
+	// Get images from Installation.KnownImages, see UpdateCollectorSpecsWithRegistryData
+	images := []string{}
+	for _, image := range kotskinds.Installation.Spec.KnownImages {
+		images = append(images, image.Image)
+	}
+
+	preflight.Spec.Collectors = append(preflight.Spec.Collectors, &troubleshootv1beta2.Collect{
+		Registry: &troubleshootv1beta2.Registry{
+			Images: images,
+		},
+	})
+	preflight.Spec.Analyzers = append(preflight.Spec.Analyzers, &troubleshootv1beta2.Analyze{
+		Registy: &troubleshootv1beta2.RegistryAnalyze{
+			AnalyzeMeta: troubleshootv1beta2.AnalyzeMeta{
```

Comment:
@AmberAlston need a review for this text in this analyzer.  This is the last row in the screenshot attached in description.

Reply:
I'm good with the current description text. I'd like to see "Private Images" updated to "Private Registry Images Available"